### PR TITLE
Pre-launch Frontend Assembly

### PR DESCRIPTION
**Pull request by UladBohdan**
* new large meta image
* footer RSS button
* banner became properly clickable
* lang cookie https
* localize tag page title
* footer copyright paddings
* narrow sidebar
* **new layout for topic page**

### CODE REVIEW

**Comment by UladBohdan** [comment]
[rss footer link](https://trello.com/c/2qeV7FfM/468-rss-footer-link)

**Comment by Drapegnik** [comment]
[lang cookie is not being saved nor read](https://trello.com/c/IbC5XWIE/467-lang-cookie-is-not-being-saved-nor-read)

**Comment by andemerie** [comment]
[Diary Block & Page](https://trello.com/c/jlK940YT/456-diary-block-page)

**Comment by UladBohdan** [comment]
[topic page columns align](https://trello.com/c/EtIFVUWY/471-topic-page-columns-align)

**Comment by UladBohdan** [comment]
[Topic Page: filling grid row by row, not column by column](https://trello.com/c/wp2cnZ7W/419-topic-page-filling-grid-row-by-row-not-column-by-column)

**Review by Drapegnik — DISMISSED** [review]

File: components/articles/Article.js [hunk 1]
```diff
@@ -30,7 +36,13 @@ const Article = ({
     <>
       <MetaTitle title={title} type="article" />
       <MetaDescription description={subtitle} />
-      <MetaImage url={`${host}${images.page || images.horizontal}`} />
+      <MetaImage
+        url={
+          (images.page && `${host}${images.page}`) ||
+          (images.horizontal && `${host}${images.horizontal}`) ||
+          DEFAULT_IMAGE
```
Comment (Drapegnik) [inline components/articles/Article.js:43]:
```js
const imageUrl = images.page || images.horizontal;
...

url={imageUrl ? `${host}${imageUrl}` : DEFAULT_IMAGE}
```

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**